pkg/api/types.go

```diff
@@ -2154,3 +2154,32 @@ type RangeAllocation struct {
 	// a single allocated address (the fifth bit on CIDR 10.0.0.0/8 is 10.0.0.4).
 	Data []byte `json:"data"`
 }
+
+// LockSpec is the specification of a resource lock.
+type LockSpec struct {
```

Comment:
It's often useful for the aquirer to shove data into the lease record. Can we add something like a map[string][]byte called AcquireInfo that could be presented by the Leaser at acquire time and renew time?